docs/testing.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Testing"
+date: 2018-05-02T00:00:00+00:00
+weight: 50
+geekdocRepo: https://github.com/owncloud/ocis-hello
+geekdocEditPath: edit/master/docs
+geekdocFilePath: testing.md
+---
+This repository provides a general guideline for creating tests for the ocis extensions. The tests can be written in various levels from unit, integration, and end-to-end. While it is not essential to write tests on all these levels as it can be redundant in some cases. This repository provides a reference for all levels of tests.
```

Comment:
```suggestion
This repository provides a general guideline for creating tests for the ocis extensions. The tests can be written in various levels from unit, integration, and end-to-end. It is not essential to write tests on all these levels as it can be redundant in some cases. This repository provides a reference for all levels of tests.
```